<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>UDTClass</class>
 <widget class="QMainWindow" name="UDTClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>519</width>
    <height>351</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>UDT</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>发送端</string>
       </attribute>
       <layout class="QGridLayout" name="gridLayout">
        <item row="1" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>初始化服务器:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sender_init">
            <property name="text">
             <string>初始化</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="3" column="0">
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLineEdit" name="line_sender">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sender_check">
            <property name="text">
             <string>浏览</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="btn_sender_send">
            <property name="text">
             <string>发送</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>选择文件发送</string>
          </property>
         </widget>
        </item>
        <item row="4" column="0">
         <widget class="QPlainTextEdit" name="text_sender"/>
        </item>
        <item row="0" column="0">
         <widget class="QLineEdit" name="line_send_ip">
          <property name="text">
           <string>127.0.0.1</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLineEdit" name="line_send_port">
          <property name="text">
           <string>8450</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>接收端</string>
       </attribute>
       <widget class="QPlainTextEdit" name="text_rec">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>90</y>
          <width>451</width>
          <height>211</height>
         </rect>
        </property>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>30</y>
          <width>457</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="line_ip"/>
         </item>
         <item>
          <widget class="QLineEdit" name="line_port"/>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>初始化服务器连接</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_rec_init">
           <property name="text">
            <string>初始化</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="layoutWidget">
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>60</y>
          <width>451</width>
          <height>25</height>
         </rect>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>选择输出位置</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="line_rec_location"/>
         </item>
         <item>
          <widget class="QPushButton" name="btn_rec_location">
           <property name="text">
            <string>确定</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="btn_rec_listen">
           <property name="text">
            <string>监听文件</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="UDT.qrc"/>
 </resources>
 <connections/>
</ui>
